tests/controllers/user.spec.js

```diff
@@ -0,0 +1,44 @@
+const user = require('../../controllers/user');
+const mongoose = require('mongoose');
+let testUserId = '';
+
+beforeAll(async () => {
+  const url =  process.env.MONGODB_URI || "mongodb://localhost/anonteamfeedback";
```

Comment:
We should really be using a separate test database for these tests. So for the local environment, we would use something like `mongodb://localhost/anonteamfeedback-test` for our local testing and `process.env.MONGODB_URI_TEST`for stating and production. That way we can safely clean up those databases without polluting the production instances with data in case your tests don't clean up the data properly. We'll probably have to create those test databases in mLabs and attach them to Heroku.

Reply:
Agreed, will make that change